server/translations.json

```
@@ -351,6 +392,86 @@
     "white-out": "Корректор",
     "zoom": "Лупа"
   },
+  "sw": {
+    "board_name_placeholder": "Jina la ubao mweupe ...",
+    "circle": "mduara",
+    "click_to_toggle": "Bonyeza kugeuza",
+    "click_to_zoom": "Bonyeza kuvuta ndani \ n Shikilia kitufe cha kuhama na ubofye ili kukuza",
+    "collaborative_whiteboard": "White board ya kushirikiana",
+    "color": "rangi",
+    "configuration": "usanidi",
+    "create_private_board": "Unda ubao mweupe wa faragha",
+    "ellipse": "ellipse",
+    "eraser": "eraser",
+    "grid": "gridi",
+    "hand": "mkono",
+    "download": "Pakua",
+    "index_title": "Karibu kwenye WBO!",
+    "introduction_paragraph": "WBO ni <a href = \" https: //github.com/lovasoa/whitebophir \ "title = \" Huru kwa maana ya kusema bure, sio bia ya bure. Programu hii imechapishwa chini ya leseni ya AGPL. Yaliyomo yamehifadhiwa. Inaweza kutumika kwa matumizi anuwai, kama sanaa, burudani, kubuni, kufundisha na kufundisha. ",
```

```suggestion
    "introduction_paragraph": "WBO ni <a href=\"https: //github.com/lovasoa/whitebophir\" title=\"Huru kwa maana ya kusema bure, sio bia ya bure. Programu hii imechapishwa chini ya leseni ya AGPL. Yaliyomo yamehifadhiwa. Inaweza kutumika kwa matumizi anuwai, kama sanaa, burudani, kubuni, kufundisha na kufundisha.",
```

The `<a` tag opens and never closes in this translation